app/src/androidTest/java/de/grobox/transportr/networks/PickTransportNetworkActivityTest.kt

```diff
@@ -73,14 +73,9 @@ class PickTransportNetworkActivityTest : ScreengrabTest() {
                 .check(matches(withText(R.string.pick_network_first_run)))
         makeScreenshot("1_FirstStart")
 
-        // hack to find region position in list
-        val regionList = ArrayList(EnumSet.allOf(Region::class.java))
-        val context = InstrumentationRegistry.getTargetContext()
-        Collections.sort(regionList) { r1, r2 -> context.getString(r1.getName()).compareTo(context.getString(r2.getName())) }
-
```

Comment:
I could re-include the scrolling code if you like, searching for and opening Europe, then Germany and then Deutsche Bahn?

Reply:
Yes, I think that would be the safest thing to do.

Reply:
Hmmm, while working on it, I thought it would be much easier to reimplement the scrolling function after changing the iterations method (probably as suggested in another PR?). For now, it would be necessary to copy the code from `PickTransportNetworkActivity` which I think is not the best option.

Btw, I've searched for this espresso testing functions and found [this documentation](https://developer.android.com/reference/android/support/test/espresso/contrib/RecyclerViewActions.html#actionOnItem(org.hamcrest.Matcher%3Candroid.view.View%3E,%20android.support.test.espresso.ViewAction)) which states:

> Performs a ViewAction on a view matched by viewHolderMatcher.
> 1. Scroll Recycler View to the view matched by itemViewMatcher
> 2. Perform an action on the matched view

So I don't quite understand why it doesn't work, I've tested it with another network which indeed was out of the display and it crashes. Would it be okay to handle this scrolling thing in the same PR suggested [here](https://github.com/grote/Transportr/pull/395#discussion_r168926895)?

Reply:
I would prefer not to merge anything which breaks tests.

Reply:
Fixed the test while introducing a `subRegions` list for `ParentRegion` (so, for `Continent` and `Country`). It's now using the same hack as before.